build.gradle

```diff
@@ -19,10 +19,20 @@ apply plugin: 'com.moowork.node'
 apply plugin: "org.sonarqube"
 apply plugin: 'project-report' // adds dependencyReport and htmlDependencyReport tasks
 
-if (file('localSettings.gradle').exists()) {
-	apply from: 'localSettings.gradle'
+// if localSettings.gradle does not exist lets create one with some sensible defaults
+if (!file('$projectDir/localSettings.gradle').exists()) {
+	new File('localSettings.gradle').text ="""
+run.environment("DB_DRIVER","sqlserver")
+run.environment("ANET_DB_SERVER","localhost")
+run.environment("ANET_DB_NAME","testAnet")
+run.environment("ANET_DB_USERNAME","anetUser")
+run.environment("ANET_DB_PASSWORD","P@ssw0rd")
+run.environment("ANET_DB_PASSWORD","P@ssw0rd")
+run.environment("ANET_DB_PORT","1433")"""
```

Comment:
For readability I'd add a newline here:
```
run.environment("ANET_DB_PORT","1433")
"""
```
and I prefer a space before each comma.